com.transmissionbt.Transmission.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "com.transmissionbt.Transmission",
+    "branch": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "transmission-gtk",
+    "rename-desktop-file": "transmission-gtk.desktop",
+    "rename-icon": "transmission",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=session-bus",
```

Comment:
Is running multiple instances with separate configs that common? I'd rather not grant it access to session bus unless it is popular among users.